pkg/core/manifest.go

```diff
@@ -149,3 +160,19 @@ func checkCompleteness(m Manifest) error {
 	}
 	return fmt.Errorf("Manifest is incomplete: %s array missing: %#v", omission, m)
 }
+
+func checkResource(resource string) error {
+	u, err := url.Parse(resource)
+	if err != nil {
+		return err
+	}
+	if u.Scheme == "http" || u.Scheme == "https" || (u.Scheme == "" && !filepath.IsAbs(u.Path)) {
+		return nil
+	}
+
+	if u.Scheme == "" {
```

Comment:
this can be removed, the last return will then cover file paths and unsupported schemes

Reply:
although the error message will be a bit less specific